app/presenters/transaction_file_body.presenter.js

```diff
@@ -52,22 +52,24 @@ class TransactionFileBodyPresenter extends BasePresenter {
       col31: this._blankIfCompensationChargeOrMinimumCharge(data.lineAttr6, data),
       col32: this._blankIfCompensationChargeOrMinimumCharge(data.lineAttr7, data),
       col33: this._blankIfCompensationChargeOrMinimumCharge(data.lineAttr8, data),
-      col34: this._blankIfCompensationChargeOrMinimumCharge(data.lineAttr9, data),
-      col35: this._blankIfCompensationChargeOrMinimumCharge(data.lineAttr10, data),
+      col34: this._blankIfCompensationChargeOrMinimumCharge(this._cleanseNull(data.lineAttr9), data),
+      col35: this._blankIfCompensationChargeOrMinimumCharge(this._cleanseNull(data.lineAttr10), data),
       col36: '',
       col37: '',
       col38: this._blankIfNotCompensationCharge(data.lineAttr13, data),
       col39: this._blankIfNotCompensationCharge(data.lineAttr14, data),
       col40: '',
       col41: '1',
       col42: 'Each',
-      col43: data.chargeValue
+      col43: this._signedCreditValue(data.chargeValue, data.chargeCredit)
```

Comment:
If my previous suggestion is taken on this would become

```suggestion
      col43: this._reverseSignOfValue(data.chargeValue)
```